packaging/docker/entrypoint.py

```diff
@@ -0,0 +1,136 @@
+    sleep_secs = os.environ.get('SLEEP_SECS')
+    max_size_in_mb_all_logs = os.environ.get('MAX_SIZE_IN_MB_ALL_LOGS')
+    max_logfiles_num = os.environ.get('MAX_LOGFILES_NUM')
+
+    return {
+        'loglevel': loglevel,
+        'logfromothermodules': logfromothermodules,
+        'sleep_secs': sleep_secs,
+        'max_logfiles_num': max_logfiles_num,
+        'max_size_in_mb_all_logs': max_size_in_mb_all_logs
+    }
+
+
+def load_config():
+    env_config = load_config_from_env()
+    file_config = load_config_from_file()
+
+    if file_config is not None:
+        logger.info('loading config from file')
+        loglevel = file_config.get('loglevel')
+        logfromothermodules = file_config.get('logfromothermodules')
+        sleep_secs = file_config.get('sleep-secs')
+        max_logfiles_num = file_config.get('max_logfiles_num')
+        max_size_in_mb_all_logs = file_config.get('max_size_in_mb_all_logs')
+    else:
+        logger.info('loading config from env')
+        loglevel = env_config.get('loglevel')
+        logfromothermodules = env_config.get('logfromothermodules')
+        sleep_secs = env_config.get('sleep_secs')
+        max_logfiles_num = env_config.get('max_logfiles_num')
+        max_size_in_mb_all_logs = env_config.get('max_size_in_mb_all_logs')
+
+    args = [
+        '--data-dir',
+        '/data',
+        '--logfile',
+        '/logs/rotki.log',
+        '--loglevel',
+        loglevel if loglevel is not None else DEFAULT_LOG_LEVEL
+    ]
+
+    if logfromothermodules is not None and logfromothermodules is True:
+        args.append('--logfromothermodules')
+
+    if sleep_secs is not None:
+        args.append('--sleep-secs')
+        args.append(str(sleep_secs))
+
+    if max_logfiles_num is not None:
+        args.append('--max-logfiles-num')
+        args.append(str(max_logfiles_num))
+
+    if max_size_in_mb_all_logs is not None:
+        args.append('--max-size-in-mb-all-logs')
+        args.append(str(max_size_in_mb_all_logs))
+
+    return args
+
+
+base_args = [
+    '/usr/sbin/rotki',
+    '--rest-api-port',
+    '4242',
+    '--websockets-api-port',
+    '4243',
+    '--api-cors',
+    'http://localhost:*/*,app://.',
+    '--api-host',
+    '0.0.0.0',
+]
+
+config_args = load_config()
+cmd = base_args + config_args
+
+logger.info('starting rotki backend')
+
+rotki = subprocess.Popen(cmd)
+
+if rotki.returncode == 1:
+    logger.error('Failed to start rotki')
+    exit(1)
+
+logger.info('starting nginx')
+
+nginx = subprocess.Popen('nginx -g "daemon off;"', shell=True)
+
+if nginx.returncode == 1:
+    logger.error('Failed to start nginx')
+    exit(1)
+
+time.sleep(60)
+
+while True:
+    if rotki.poll() is not None:
+        logger.error('rotki has terminated exiting')
+        exit(1)
+
+    if nginx.poll() is not None:
+        logger.error('nginx was not running')
+        exit(1)
+
+    logger.info('OK: processes still running')
+    time.sleep(60)
```

Comment:
Also you can avoid the `time.sleep(60)` in 124 by putting this at the beginning of the loop